pages/07.Channels/01.Marketing Messages/03.SMS/docs.en.md

```diff
@@ -33,8 +34,27 @@ Before you start to send text messages from your Mautic instance you need to con
 
 The new Text message will be pre-selected so you can save the *Send Text Message* action as well. You can use the action in your Campaign dripflow.
 
+## Tracking Replies and Unsubscribes
+
+Contacts can unsubscribe from your SMS messags by replying with the word "Stop" to your SMS, Once Mautic recive this SMS that specific contact will be marked as Do not Contact and will not be contacted again.
```

Comment:
A couple of typos, and I believe the DNC is set per-channel so we should be clear that they will not be contacted by SMS but may still continue to be contacted via other channels (that is my understanding of how it works, at least).

```suggestion
Contacts can unsubscribe from your SMS messages by replying with the word "Stop" to your SMS. Once Mautic receives this SMS, that specific contact will be marked as Do Not Contact (DNC) and will not be contacted by SMS again.
```